src/utils/bytesParsers/i18n/ru.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
why?

Reply:
Time ago decided not to support /ru localization and if change en keysets, remove ru version

Reply:
As far as I remember we decided to support ru localization